src/patternfly/components/Label/label.scss

```diff
@@ -203,6 +203,7 @@
   --#{$label}--m-compact--PaddingInlineStart: var(--pf-t--global--spacer--sm);
   --#{$label}--m-compact--FontSize: var(--pf-t--global--font--size--body--sm);
   --#{$label}--m-compact--m-editable--TextUnderlineOffset: #{pf-size-prem(1px)};
+  --#{$label}--m-compact--MinWidth: calc(var(--#{$label}--m-compact--FontSize) * var(--pf-t--global--font--line-height--body));
```

Comment:
Should compact use the same formula, adding on the block paddings just in case someone adjusts those variables to something other than zero?